docs/releases/changelog-dev.md

```diff
@@ -57,6 +57,28 @@
 
   [#762](https://github.com/qilimanjaro-tech/qililab/pull/762)
 
+- Added `filter` argument inside the qua config file compilation from runcards with qm clusters. This is an optional element for distorsion filters that includes feedforward and feedback, two distorion lists for distorsion compensation and fields in qua config filter. These filters are calibrated and then introduced as compensation for the distorsions of the pulses from external sources such as Bias T. The runcard now might include the new filters (optional):
+
+  Example:
+    ```
+  instruments:
```

Comment:
Watch out, all the example needs to be **indented**!

Not only the line I added in the commit, sorry for the confusion!

Reply:
Done!